src/cljs/athens/views/blocks/drop_area_indicator.cljs

```diff
@@ -1,6 +1,6 @@
 (ns athens.views.blocks.drop-area-indicator
   (:require
-    [athens.style :as style :refer [OPACITIES]]
+    [athens.style :refer [color OPACITIES]]
```

Comment:
Reducing the requires here

Reply:
It's more readable in code when we use `:as alias`, you know straight away that it's from outside local namespace.